### PR TITLE
[rosa hcp] to add ap-southeast-2 and ap-southeast-4 regions

### PR DESCRIPTION
**Pull request by sdminonne**
<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][OCPBUGS#<jira-issue-id>][OSDOCS#<jira-issue-id>]: 

<!--- If your changes apply to the latest release and/or in-development version of OpenShift, open your PR against the `main` branch.

### CODE REVIEW

File: modules/rosa-sdpolicy-am-regions-az.adoc [hunk 1]
```diff
@@ -37,9 +37,10 @@ ifndef::rosa-with-hcp[]
 * ap-south-1 (Mumbai)
 * ap-south-2 (Hyderabad, AWS opt-in required)
 * ap-southeast-1 (Singapore)
-* ap-southeast-2 (Sydney)
 endif::rosa-with-hcp[]
+* ap-southeast-2 (Sydney)
 * ap-southeast-3 (Jakarta, AWS opt-in required)
+* ap-southeast-4 (Melbourne, AWS opt-in required)
 ifndef::rosa-with-hcp[]
```
Comment (EricPonvelle) [inline modules/rosa-sdpolicy-am-regions-az.adoc:44]: There's a duplicated zone here on the ROSA Classic side

image

That isn't present on ROSA with HCP:

image

**Review by EricPonvelle — COMMENTED** [review]
Small fix needed for the ROSA Classic regions but otherwise looks good.

**Comment by sdminonne** [comment]
@EricPonvelle PTAL

**Comment by sdminonne** [comment]
@EricPonvelle squashed and commit message updated

**Comment by sdminonne** [comment]
Sorry didn't rebase

**Pull request merged**